i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-database-sqldatabase.md

```diff
@@ -390,28 +393,25 @@ que le connecteur arrive bien à superviser une instance Azure en utilisant une
 telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
 
 ```bash
-/usr/lib/centreon/plugins//centreon_azure_database_sqldatabase_api.pl \
+/usr/lib/centreon/plugins/centreon_azure_database_sqldatabase_api.pl \
 	--plugin=cloud::azure::database::sqldatabase::plugin \
-	--mode=app-resources \
+	--mode=workers \
```

Comment:
not many parameters here, is that normal? the mandatory ones should be filled in, at least